CHANGELOG.md

```diff
@@ -171,8 +171,13 @@ purple|#2e358b|#4c2c92
 red|#b10e1e|#d4351c
 yellow|#ffbf47|#ffdd00
 green|#006435|#00703c
+grey-1|#6f777b|#6f777b
```

Comment:
Looks good 👍 
Because we've framed this as 'change these colours', I'd probably remove `grey-1`. Otherwise users might think we've made a mistake and meant to include a different #rgb value.

Reply:
ok 👍